charter.md

```diff
@@ -0,0 +1,41 @@
+# Interop Team Charter
+
+The Interop Team aims to make the web more interoperable in key areas, prioritized by web developer and user needs. This team is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt), the main test suite for the web platform.
+
+The team is responsible for maintaining [Compat 2021](https://wpt.fyi/compat2021), [Interop 2022](https://wpt.fyi/interop-2022), and defining future interoperability efforts and metrics.
+
+## Scope
+
+The team works by prioritizing specific web platform features. The minimum requirements for a feature to be in scope are:
+
+- The feature is defined by a specification from the [IETF](https://www.ietf.org/), [Khronos Group](https://www.khronos.org/), [TC39](https://tc39.es/), [W3C](https://www.w3.org/), or [WHATWG](https://whatwg.org/). Other specifications can be considered on a case-by-case basis.
+- The feature can be tested in [web-platform-tests](https://github.com/web-platform-tests/wpt). Other test suites can be considered on a case-by-case basis.
```

Comment:
```suggestion
- The feature can be **automatically** tested in [web-platform-tests](https://github.com/web-platform-tests/wpt). Other test suites can be considered on a case-by-case basis.
```

Added automatically. Many tests in web-platform-tests currently requires manual testing. We want to avoid this, to not burden ourselves into having to run manual tests. 

It's the second time that the charter introduces a case-by-case basis. While I understand that this is a way to give a bit of grease to the process, this is also an opportunity for having endless discussions. 

**TODO**: Open an issue about this case by case notion. What do we mean? What are the precedents? Was it needed? And Why? Was the group too constrained? -> #123

Reply:
Maybe we need room in the process on how to propose a new test suite, or new standard organization.

Reply:
I actually think we should leave the door open to manual tests, but we could leave that as one of the case-by-case situations. For example, if we found that OKLab gradients have some serious interop issue, but WPT is not yet able to test this for some reason, then having a few visual tests that we test once a month could be acceptable, if that's what makes the difference for improving interop.

Reply:
+1 to @foolip on leaving things open to manual / non-WPT testing. If having the possibility truly ends up being a problem, you can restrict it then, but there are many things (e.g. Media Queries) that WPT is not currently good at testing that are nonetheless worth reaching interop on.

Reply:
I'd much rather we had _something_ in here about automated tests, and perhaps make it clear that the "case-by-case basis" includes manual tests in WPT?

Reply:
How about this?

> The feature can be tested by automated tests in [web-platform-tests](https://github.com/web-platform-tests/wpt). Manual tests and other test suites can be considered on a case-by-case basis.

Reply:
I've pushed this edit.